### DISCUSSION TITLE
Blobbing the food emojis

### DISCUSSION THREAD

**Discussion by JCLemme** [C1710/blobmoji#68] [Style]
I've started converting some of the new food emojis to blob style. Kinda scary because I actually have to draw these - I'm not just playing build-a-blob with copy and paste. Here's a first draft of pumpkin pie - putting it in Issues since we're nowhere near prime time ready:
![newpie](https://user-images.githubusercontent.com/22945718/76390890-a1b2d880-6344-11ea-98d5-4685789091be.png)
Compare this with the blob cake and the modern Noto pie:
![cake](https://user-images.githubusercontent.com/22945718/76390931-b42d1200-6344-11ea-97f6-c68d91436193.png)
![oldpie](https://user-images.githubusercontent.com/22945718/76390933-b5f6d580-6344-11ea-8ba4-15f43dc32b33.png)
Outline was loosely traced over the cake, colors copied from the jack-o-lantern emoji. I'm mostly happy with the pie but the whipped cream needs work. Any thoughts?

**Comment by Oymate** [comment]
It looks a bit gooey and too weird red

**Comment by ann4belle** [comment]
Might be out of scope, but there's no harm in forking the repo and submitting a PR.

**Comment by Oymate** [comment]
Maybe try the pie colours on the current shape?
